upstream-community-operators/percona-server-mongodb-operator/1.6.0/percona-server-mongodb-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,388 @@
+    can be used to instantiate a new database cluster or to scale an existing
+    database cluster. The Operator contains all necessary Kubernetes settings to
+    provide a proper and consistent Percona Server for MongoDB replica set.
+
+
+    Consult the
+    [documentation](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html/)
+    on the Percona Kubernetes Operator for Percona Server for MongoDB for
+    complete details on capabilities and options.
+
+
+    ### Supported Features
+
+
+    * **Scale Your Cluster** - change the `size` parameter to [add or remove
+    members](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/scaling.html)
+    of the replica set. Three is the minimum recommended size for a functioning
+    replica set.
+
+
+    * **Add Monitoring** - [Percona Monitoring and Management (PMM) can be
+    easily
+    deployed](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/monitoring.html)
+    to monitor your Percona Server for MongoDB replica set(s). The recommended
+    installation process uses Helm, the package manager for Kubernetes.
+
+
+    * **Automate Your Backups** - [configure automatic
+    backups](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/backups.html)
+    to run on a scheduled basis or run an on-demand backup at any time. Backups
+    are performed using Percona Backup for MongoDB (PBM) and can be stored on
+    local PVs or in any S3-compatible cloud storage provider.
+
+
+    ### Common Configurations
+
+
+    * **Set Member as Arbiter** - [Set up a replica set which contains an
+    arbiter](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/arbiter.html),
+    which participates in elections but does not store any data.  This reduces
+    storage costs while helping maintain replica set integrity.
+
+
+    * **Expose Members Outside K8S** - [by appropriately configuring the
+    ServiceType](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/expose.html)
+    you can expose replica set members outside of Kubernetes or provide
+    statically assigned IP addresses.
+
+
+    * **Utilize Local Storage Options** - [with support for Local Storage you
+    can mount existing data
+    directories](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/storage.html)
+    into your replica set managed by Kubernetes or utilize high performance
+    hardware for local storage rather than network storage for your database.
+
+
+    ### Before You Start
+
+
+    Add the PSMDB user `Secret` to Kubernetes. User information must be placed
+    in the data section of the `secrets.yaml`
+
+    file with Base64-encoded logins and passwords for the user accounts.
+
+
+    Below is a sample `secrets.yaml` file for the correct formatting.
+
+
+    ```
+
+    apiVersion: v1
+
+    kind: Secret
+
+    metadata:
+      name: my-cluster-name-secrets
+    type: Opaque
+
+    data:
```

Comment:
This should be changed to `stringData` and new list of plain text passwords.

Reply:
done